community/cypher/cypher/src/test/scala/org/neo4j/cypher/NewPlannerTestSupport.scala

```diff
@@ -168,7 +168,8 @@ trait NewPlannerTestSupport extends CypherTestSupport {
 
     assertResultsAreSame(interpretedResult, compiledResult, queryText, "Diverging results between interpreted and compiled runtime")
     assertResultsAreSame(ruleResult, interpretedResult, queryText, "Diverging results between rule planner and interpreted runtime")
-
+    ruleResult.close()
+    interpretedResult.close()
```

Comment:
These changes are not really related to the PR, right? Perhaps they really belong in the #4550 PR?